src/components/ChangePassword/ChangePassword.tsx

```diff
@@ -82,9 +83,13 @@ const ChangePassword: React.FC<Props> = ({
       oldPassword: yup
         .string()
         .required(t('forms.validation.required'))
-        .min(passwordComplexity.minLength, t('forms.validation.minLength', { var: passwordComplexity.minLength }))
+        .min(4, t('forms.validation.minLength', { var: 4 }))
+        .max(32, t('forms.validation.maxLength', { var: 32 })),
+      newPassword: yup
+        .string()
+        .required(t('forms.validation.required'))
+        .min(12, t('forms.validation.minLength', { var: 12 }))
         .max(32, t('forms.validation.maxLength', { var: 32 })),
```

Comment:
```suggestion
        .max(64, t('forms.validation.maxLength', { var: 64 })),
      newPassword: yup
        .string()
        .required(t('forms.validation.required'))
        .min(12, t('forms.validation.minLength', { var: 12 }))
        .max(64, t('forms.validation.maxLength', { var: 64 })),
```

<img width="1022" height="344" alt="Image" src="https://github.com/user-attachments/assets/42561aa8-8d56-4049-bf91-2a2aa1b93a40" />